lib/stdio/input.js

```diff
@@ -69,7 +24,6 @@ const handleInputFileOption = (stdioStream, index, inputFile, input) => {
 
 	const optionName = 'inputFile';
 	validateInputOption(stdioStream.value, optionName);
-	validateRegularUrl(inputFile, optionName);
```

Comment:
This is now handled by the `validateFileStdio()` method above, which has been slightly reworked.